tests/unit/torch/utils/test_selection_utils.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from merlin.models.torch.utils.selection_utils import Selectable, select_schema, selection_name
+from merlin.schema import ColumnSchema, Schema, Tags
+
+
+class Test_select_schema:
```

Comment:
For consistency with the other test classes, perhaps we should align on a similar style across our tests for the naming here. e.g. CamelCase like in the `test_router.py`?

Reply:
So far I did camel-case for classes and snake-case for functions. Do you think it's better to use camel-case for all?

Reply:
`Test_select_schema` -> `TestSelectSchema`?